app/api/ltc.py

```diff
@@ -241,7 +274,7 @@ def do_aggregate_outbreaks(df):
 
     # TODO: if more than FL needs special treatment before aggregating outbreaks, factor this out
     # into something nicer
-    state = set(df['State']).pop()
+    state = {x for x in set(df['State']) if x==x}.pop()
```

Comment:
sorry I got confused by this, what was the context where this came up?

Reply:
when I ran this on the FL Facility sheet, `set(df['State']).pop()` was returning nan so I threw this in to only include non-nan elements in the set and then pop